lib/pool.js

```diff
@@ -27,21 +27,19 @@ function _addBackoff(factory) {
 
   factory.create = function __run() {
 
-    return originalCreateMethod().catch((e) =>
+    return originalCreateMethod().catch(function ___handleErrorInCreate(e) {
 
-      new Bluebird(function ___backoffRunResolver(resolve, reject) {
-        const wait = backoff.next()
+      const wait = backoff.next()
```

Comment:
however, there is only one `backoff` object created per instance of the resource factory object.  Therefore, each call to `factory.create` will refer to the same `backoff` object.  If you run `DEBUG=kuss:pool* make test` you will see the wait time increasing.